tests/conftest.py

```diff
@@ -0,0 +1,56 @@
+import os
+import os.path
+
+import pytest
+
+from pytest_bdd import given
+from pytest_bdd import parsers
+from pytest_bdd import then
+from pytest_bdd import when
+
+from utils.helper import run_ansible_playbook
+
+
+@pytest.fixture(scope="session")
+def inventory():
+    inventory_file = os.environ.get('ANSIBLE_INVENTORY')
+    assert os.path.exists(inventory_file)
+    if not os.path.exists(inventory_file):
```

Comment:
How would this work when there's an `assert` before?

Reply:
oups